packages/office-ui-fabric-react/src/components/Pivot/Pivot.Props.ts

```diff
@@ -50,6 +50,18 @@ export interface IPivotProps extends React.Props<Pivot> {
    */
   linkFormat?: PivotLinkFormat;
 
+  /**
+   * Specify whether to skip rendering the tabpanel with the content of the selected tab.
+   * Use this prop if you plan to separately render the tab content
+   * and don't want to leave an empty tabpanel in the page that may confuse Screen Readers.
+   */
+  headersOnly?: boolean;
```

Comment:
It feels like ideally we would split the component into a `PivotItemBar` that only renders `PivotItem`s and a `Pivot` that uses `PivotItemBar` to display the header and also shows the body content. Did you think about that?